IPython/lib/latextools.py

```diff
@@ -15,7 +14,12 @@
 from traitlets.config import get_config
 from traitlets.config.configurable import SingletonConfigurable
 from traitlets import List, Bool, Unicode
-from IPython.utils.py3compat import cast_unicode, cast_unicode_py2 as u
+from IPython.utils.py3compat import cast_unicode, cast_unicode_py2 as u, PY3
+
+if PY3:
```

Comment:
I'm thinking it's easier to cleanup py2 compat code (when we drop it) if it's explicit. I know EAFP. But here I'm leaning more toward "Explicit is better than implicit". I'm alway doubting when I see a `try/except ImportError` for the reason.